assets/queries/terraform/aws/cmk_rotation_disabled/query.rego

```diff
@@ -17,21 +19,80 @@ CxPolicy[result] {
 	}
 }
 
+
 CxPolicy[result] {
 	resource := input.document[i].resource.aws_kms_key[name]
 
 	not key_set_to_false(resource)
-	resource.enable_key_rotation == false
+    resource.enable_key_rotation == true
+    common_lib.valid_key(resource, "customer_master_key_spec")
+    not customer_master_key_spec_set_to_symmetric(resource)
 
 	result := {
 		"documentId": input.document[i].id,
-		"searchKey": sprintf("aws_kms_key[%s].enable_key_rotation", [name]),
-		"issueType": "IncorrectValue",
+		"searchKey": sprintf("aws_kms_key[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("aws_kms_key[%s].enable_key_rotation is set to false", [name]),
+		"keyActualValue": sprintf("aws_kms_key[%s].enable_key_rotation is true", [name]),
+	}
+}
+
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_kms_key[name]
+
+	not key_set_to_false(resource)
+    resource.enable_key_rotation == false
+    customer_master_key_spec_set_to_symmetric(resource)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_kms_key[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("aws_kms_key[%s].enable_key_rotation is set to true", [name]),
+		"keyActualValue": sprintf("aws_kms_key[%s].enable_key_rotation is false", [name]),
+	}
+}
+
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_kms_key[name]
+
+	not key_set_to_false(resource)
+    not common_lib.valid_key(resource, "enable_key_rotation")
+    not common_lib.valid_key(resource, "customer_master_key_spec")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_kms_key[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("aws_kms_key[%s].enable_key_rotation is set to true", [name]),
+		"keyActualValue": sprintf("aws_kms_key[%s].enable_key_rotation is undefined", [name]),
+	}
+}
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_kms_key[name]
+
+	not key_set_to_false(resource)
+    resource.enable_key_rotation == false
+    not common_lib.valid_key(resource, "customer_master_key_spec")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_kms_key[%s]", [name]),
+		"issueType": "MissingAttribute",
 		"keyExpectedValue": sprintf("aws_kms_key[%s].enable_key_rotation is set to true", [name]),
 		"keyActualValue": sprintf("aws_kms_key[%s].enable_key_rotation is set to false", [name]),
 	}
 }
```

Comment:
```suggestion
```

Reply:
@LupovichRan, this request change is missing. Can you delete this policy, please?
Thank you so much for being so quick! 😊